stripper/ze_forhidden_slicoo_b10_5.cfg

```diff
@@ -0,0 +1,574 @@
+;==================================================
+;              Stripper Made by Koen
+;           Found a bug? Contact me pls
+;            Steam: /id/TheProKoen
+;          Discord: TheProKoen#4977
+;==================================================
+
+;Add Stripper Credits
+modify:
+{
+    match:
+    {
+        "classname" "logic_auto"
+        "hammerid" "57143"
+    }
+    insert:
+    {
+        "OnMapSpawn" "cmd,Command,say ## Stripper by TheProKoen for GFL Maptest ##,5,1"
```

Comment:
Not sure why it's necessary to specify this as for a map test? This would of course be visible still after testing.